apps/common/schemas.py

```diff
@@ -58,3 +62,73 @@ class FullNameSchema(BaseOutSchema):
     last_name: str
     first_name: str
     middle_name: str = None
+
+
+class MessageSchema(BaseModel):
+    recipients: List[EmailStr]
+    attachments: List[Union[UploadFile, Dict, str]] = []
+    subject: str = ""
+    body: Optional[Union[str, list]] = None
+    template_body: Optional[Union[list, dict]] = None
+    html: Optional[Union[str, List, Dict]] = None
+    cc: List[EmailStr] = []
+    bcc: List[EmailStr] = []
+    reply_to: List[EmailStr] = []
+    charset: str = "utf-8"
+    subtype: Optional[str] = None
+    multipart_subtype: MultipartSubtypeEnum = MultipartSubtypeEnum.mixed
+    headers: Optional[Dict] = None
+
+    @validator("attachments")
+    def validate_file(cls, v):
```

Comment:
rename from `validate_file` to `validate_ attachments ` if you use `@validator("attachments")`

Reply:
I won't change it because of the fact that from that changing nothing changes and the fact that this code is from official packet documentation